.github/workflows/tpcds-reusable.yml

```diff
@@ -42,6 +42,22 @@ on:
         required: false
         type: string
         default: ''
+      unifflever:
+        required: false
+        type: string
+        default: ''
+      uniffleurl:
+        required: false
+        type: string
+        default: ''
+      hadoopver:
```

Comment:
for uniffle, hadoop dependency are provided, so have to install the hadoop binary